addons/handhelds/radio_ids.hpp

```diff
@@ -1,73 +1,1012 @@
-#define TF_RADIO_ID_full(baseClass,displayNameBase,index) class baseClass##_##index : baseClass \
+#define TF_RADIO_ID(baseClass,displayNameBase,index) class baseClass##_##index : baseClass \
     { \
-        displayName = displayNameBase index; \
+        displayName = 'displayNameBase index'; \
         scope = 1; \
         scopeCurator = 1; \
         tf_prototype = 0; \
         tf_radio = 1; \
         ace_arsenal_uniqueBase = baseClass; \
     };
 
+#define TF_RADIO_ID_LIST(baseClass,displayNameBase) \
```

Comment:
No. What tools are you using that are too dumb for that?

Reply:
Mikereo.... I was surprised too, but it's the way it is....

Reply:
We use Mikero on CI and it works fine